packages/hardhat/contracts/SifiV1Router01.sol

```diff
@@ -90,13 +90,15 @@ contract SifiV1Router01 is Ownable {
 
     if (amounts[1] > amountOut) {
       // Transfer positive slippage to fee address
-      payable(fees).transfer(amounts[1] - amountOut);
+      (bool sentPositiveSlippage, ) = (fees).call{value: amounts[1] - amountOut}('');
+      require(sentPositiveSlippage, 'ETH_TRANSFER_POS_SLIP_FAILED');
 
       amounts[1] = amountOut;
     }
 
     // Transfer ETH to user
-    payable(to).transfer(amounts[1]);
+    (bool sent, ) = to.call{value: amounts[1]}('');
+    require(sent, 'ETH_TRANSFER_FAILED');
```

Comment:
I suggest sticking to the standard `(bool success, )` for transactions and considering the use of custom errors over `require()`, like ` if(!success) {revert EthTransferFailed()}` with a custom `error`.